### ISSUE TITLE
Feature Suggestion: Simple editing

### ISSUE THREAD

**Issue by pondturtle** [featurecat/lizzie#485]
I have been doing a lot of analysing of my fuseki with LZ lately but a lot of time it gets really obtuse having to play it whole from the first move. 

Doing analysis or study with LZ it would really help if we could just place/erase stones without having to play them in order to analyse given position.

**Comment by alreadydone** [comment]
LZ's neural net takes the last 7 moves as the input, so if you don't feed it with a reasonable history it may not come up with reasonable analysis, so this is not so trivial to implement (same for Leela Chess).

**Comment by gogre** [comment]
@alreadydone  i'm not so sure about that; for as Henry Ford said adroitly, "History is Bunk" and, if anything, having a history would distract LZ from acute observation.  

@pondturtle As to wanting editor features, Sabaki already has them; 

and i haven't yet tried it out, but FeatureCat has a Sabaki fork in Lizzie style called Shell and Slate.  In fact, that might be such a good idea, i'm gonna check it out right now :)

**Comment by bvandenbon** [comment]
@gogre I agree that in Go you should never take decisions based on the history of the game. So, it seems at first that it is a waste of resources for LeeLa having to drag that 7 move history with it.

From what I've heard, the reason behind it has to do with resolving complicated ko positions. After all, LeeLa should know which moves are legal/illegal , and you can only do that by taking the history of the game in account.

**Comment by gogre** [comment]
i want to change my testimony: when it comes to review, history is definitely not bunk, because it tells us how we got into whatever awful mess we are in.  you can rewrite history, but you can't change the past.

as to whether any person would benefit from knowing history insofar as their next move is concerned, i still believe it's only the stones that count, not how they got there, with the exception of kos, for which a history of the entire game is needed, not just the last few moves.

i also think that would be true for a DCNN bot too, but that's not a Lizzie issue.

**Comment by hanysz** [comment]
Bumping this.  I don't know if there's been a change in LZ since it was first raised.  I can use a different SGF editor to place stones, save the SGF, then load into Lizzie, and I have no problem getting reasonable analysis results on a position with no history.  So it looks as though the principal objection to implementing this is no longer valid?

**Comment by featurecat** [comment]
You're right. It would be nice to allow this. @zsalch do you want to add this? I will add this at least by 1.0 for better analysis / sgf editing.
